dotCMS/src/integration-test/java/com/dotcms/publisher/util/DependencyManagerTest.java

```diff
@@ -0,0 +1,256 @@
+        final ContentletDataGen dataGen = new ContentletDataGen(contentType.id());
+
+        //Creates parent content
+        Contentlet blogContentParent = dataGen
+                .languageId(languageId)
+                .setProperty("title", "blogContentParent")
+                .setProperty("urlTitle", "blogContentParent")
+                .setProperty("author", "systemUser")
+                .setProperty("sysPublishDate", new Date())
+                .setProperty("body", "blogBody").next();
+
+        //Creates child content
+        final Contentlet blogContentChild = dataGen
+                .languageId(languageId)
+                .setProperty("title", "blogContentChild")
+                .setProperty("urlTitle", "blogContentChild")
+                .setProperty("author", "systemUser")
+                .setProperty("sysPublishDate", new Date())
+                .setProperty("body", "blogBody").nextPersisted();
+
+        //Adds a new relationship between both contentlets
+        final Relationship relationship = relationshipAPI.byContentType(contentType).get(0);
+
+        ContentletRelationships contentletRelationships = new ContentletRelationships(
+                blogContentParent);
+        ContentletRelationshipRecords records = contentletRelationships.new ContentletRelationshipRecords(
+                relationship, true);
+        records.setRecords(Lists.newArrayList(blogContentChild));
+        contentletRelationships.getRelationshipsRecords().add(records);
+        blogContentParent = contentletAPI.checkin(blogContentParent, contentletRelationships, null, null, user, false);
+
+        //Creates a bundle with just the child
+        createBundle(config, blogContentChild);
+
+        DependencyManager dependencyManager = new DependencyManager(user, config);
+        dependencyManager.setDependencies();
+
+        //The dependecy manager should include parent and child contentlets in the bundle
+        validateDependencies(blogContentParent, blogContentChild, relationship, dependencyManager);
+
+        //Creates a bundle with just the parent
+        createBundle(config, blogContentParent);
+
+        dependencyManager = new DependencyManager(user, config);
+        dependencyManager.setDependencies();
+
+        //The dependency manager should include parent and child contentlets in the bundle
+        validateDependencies(blogContentParent, blogContentChild, relationship, dependencyManager);
+    }
+
+    /**
+     * <b>Method to test:</b> {@link DependencyManager#setDependencies()} <p>
+     * <b>Given Scenario:</b> A bundle contains a content with a relationship <p>
+     * <b>ExpectedResult:</b> The Dependency Manager should include all related content to the bundle
+     * @throws DotSecurityException
+     * @throws DotBundleException
+     * @throws DotDataException
+     */
+    @Test
+    public void test_dependencyManager_shouldIncludeRelationships()
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5775242)